libs/core/langchain_core/messages/__init__.py

```diff
@@ -117,6 +117,81 @@ def message_chunk_to_message(chunk: BaseMessageChunk) -> BaseMessage:
     )
 
 
+MessageLikeRepresentation = Union[BaseMessage, Tuple[str, str], str]
+
+
+def _create_template_from_message_type(message_type: str, template: str) -> BaseMessage:
```

Comment:
Would want to update this naming /docstrings to not say "template"

Reply:
Oops im lazy. Also needs tests